app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,20 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors_ordered_by_name_age
+    @average_age = @movie.average_actor_age
+  end
+
+  def update
```

Comment:
I mentioned a bit of this in other comments, but we shouldn't use a `update` action when we use a `post` verb for a form, since we want to `create` something with `post`. 

The actual meat of this method is great, it's exactly what we need to do. However, it should really be in a `create` action in a controller. But that's where we need to make some design decisions.... Are we really `updating` or `creating` a Movie? No, we're creating a relationship between a movie and an actor. What if we made a new controller for that, just movie-actor stuff, called `movie_actors_controller`? Then we'd have space to put the `create` method.

Reply:
This would come into play if there were another user story that had us update a movie instance. Then, you'd be out of luck with this method since it's already being used.